docs/api/interpreter.rst

```diff
@@ -71,3 +71,16 @@ the same type you can use
     /add_to_shopping_list{"item": ["milk", "salt"]}
 
 Which corresponds to a message ``"I want to add milk and salt to my list"``.
+
+If you want to specify the intent confidence you can use
+
+.. code-block:: bash
+
+    /deny@1.1
```

Comment:
given that these are most likely probabilites, I think it would be better to use an example with a confidence `<1.0` 😉